### PR TITLE
Create settings.py

### PR DESCRIPTION
**Pull request by gujjs**

### CODE REVIEW

File: vsdkx/addon/tracking/settings.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+DEFAULT = {
+    "bidirectional_mode": True,
+    "bidirectional_threshold": 150,
+    "distance_threshold": 500,
+    "max_disappeared": 10,
+    "min_appearce": 40
```
Comment (nicolexirakia) [inline vsdkx/addon/tracking/settings.py]: this is actually too high, we should reduce it to `1`
```suggestion
    "min_appearce": 1
```

File: vsdkx/addon/tracking/settings.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+}
```
Comment (nicolexirakia) [inline vsdkx/addon/tracking/settings.py:7]: Also add the `debug_mode` attribute.
```suggestion
     "debug_mode": False
}
```

**Review by nicolexirakia — COMMENTED** [review]
@gujjs two minor things, have a look and let me know if there are any questions.

File: vsdkx/addon/tracking/settings.py (hunk 2 above)
Comment (gujjs) [reply vsdkx/addon/tracking/settings.py:7]: it's wrong place for this. debug mode is not turned on from addon settings. it should be defined in model settings and gonna be read by core. this is read by tracking addon only, which doesn't need it

File: vsdkx/addon/tracking/settings.py (hunk 1 above)
Comment (gujjs) [reply vsdkx/addon/tracking/settings.py]: also, I think that there's typo in key name

File: vsdkx/addon/tracking/settings.py [hunk 3]
```diff
@@ -0,0 +1,7 @@
+DEFAULT = {
+    "bidirectional_mode": True,
+    "bidirectional_threshold": 150,
+    "distance_threshold": 500,
+    "max_disappeared": 10,
+    "min_appearce": 1
```
Comment (gujjs) [inline vsdkx/addon/tracking/settings.py]:
```suggestion
    "min_appearance": 1
```

Comment (gujjs) [reply vsdkx/addon/tracking/settings.py]: this key had a typo by mistake

**Review by nicolexirakia — APPROVED** [review]

**Pull request merged**